### PR TITLE
Use \in consistently in README

### PR DESCRIPTION
**Pull request by thsutton**
I'm not positive, but I think all the `in`s in the judgments are the same?

Make them all pretty!

### CODE REVIEW

**Comment by thsutton** [comment]
I should probably have squashed that.

File: README.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ It is still possible to prove membership and equality in the refinement logic,
 even though the categorical judgements are inaccessible. In order to do this,
 an equality type is introduced, whose *truth* is defined specifically so as to
 coincide with the (mere) evidence of CTT's equality judgement. In practice, you
-will often see goals like `H >> M in A [ext •]`, and in this case you are
```
Comment (jonsterling) [inline README.md:110]: This one is not meant to be \in; It is perhaps an unfortunate notation, but I am using "in" for the type/proposition, and "\in" for the judgement.

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ judgement; in CTT, the rules are arranged so as to cause the old
 presuppositions to become evident by virtue of the evidence of a judgement. An
 example is the meaning of the sequent `x:A >> B(x) type` (`B` is a family
 fibred over `A`): in ETT, you must demonstrate `A type`, and then you must
-demonstrate `|x B(x) type (x in A)`. In CTT, however, the meaning of the
+demonstrate `|x B(x) type (x ∈ A)`. In CTT, however, the meaning of the
```
Comment (jonsterling) [inline README.md:67]: :heavy_check_mark:

File: README.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ The primary differences between CTT and ETT are as follows:
 
 1. ETT has four categorical judgements: `A type`, `A = B type`, `M ∈ A` and
 `M = N ∈ A`; CTT has only one categorical judgement, which is `M = N ∈ A`.
-`A = B type` is recovered as equality in a universe, and `M in A` is recovered as
+`A = B type` is recovered as equality in a universe, and `M ∈ A` is recovered as
```
Comment (jonsterling) [inline README.md:51]: :heavy_check_mark:

File: README.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ even though the categorical judgements are inaccessible. In order to do this,
 an equality type is introduced, whose *truth* is defined specifically so as to
 coincide with the (mere) evidence of CTT's equality judgement. In practice, you
 will often see goals like `H >> M in A [ext •]`, and in this case you are
-proving the proposition `M in A` (which is an abbreviation for `M = M in A`).
+proving the proposition `M ∈ A` (which is an abbreviation for `M = M ∈ A`).
 Based on the meaning explanation for the refinement logic, you are indirectly
```
Comment (jonsterling) [inline README.md:112]: Ah, here is another one that is meant to be the proposition, not the judgement.

Comment (thsutton) [inline README.md:112]: Are they both the prop?

Comment (jonsterling) [inline README.md:112]: Yes.

**Comment by thsutton** [comment]
#herpaderp I have corrected my hypercorrecting. Now to go and read this more carefully so I more properly understand the distinction between propositions and judgements.

**Comment by thsutton** [comment]
Squashed.

**Comment by jonsterling** [comment]
Thank you! BTW, if the difference between propositions and judgements is not evident to you, it is my fault, not yours!

Briefly, judgements are the more primitive thing. A judgement is something that you come to know at a particular time, and when you have come to know it (by means of constructing some evidence for it in your mind), we say that it is evident. In order to introduce a form of judgement into a theory, you must give it a meaning explanation, which is to say, What does it mean to know this judgement? Or, phrased otherwise, What counts as evidence of this judgement?

On the other hand, a proposition (or type, or set) is the subject of certain judgements, including `P prop`, `P true`, and `M \in P`. In general, we speak of the _evidence of a judgement_ and the _truth of a proposition_.

**Pull request merged**